### PR TITLE
[NetAppFiles] Anf 5207 Bugfix - add missing snapshot restore function

### PR DESCRIPTION
**Pull request by audunn**
**Description**  

This is a bug fix for an issue where a missing features/functions missing from CLI this blocked users from creating volumes from snapshots. 

**Testing Guide**  

Add a snapshot id as a parameter to create volume to create a volume from existing snapshot.
netappfiles volume create --resource-group %s --account-name accontName --pool-name poolName --volume-name volumeName -l location --service-level 'Premium' --usage-threshold 100 --file-path path --vnet vNet --subnet subnetId --snapshot-id snapshotId

**History Notes**  

[NetAppFiles] az volume create: Adds snapshot-id as a parameter to create volume this will allow users to create a volume from existing snapshot.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
add to S170

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "lockfileVersion": 1
```
Comment (haroldrandom) [inline package-lock.json:2]: What's the purpose of adding this file?

File: src/azure-cli/HISTORY.rst [hunk 2]
```diff
@@ -313,7 +313,7 @@ Release History
 * Fix IcM179210086: unable to create custom metric alert for their Application Insights metric
 
 **NetAppFiles**
-
+* az volume create: Adds snapshot-id as a parameter to create volume this will allow users to create a volume from existing snapshot.
```
Comment (haroldrandom) [inline src/azure-cli/HISTORY.rst:316]: Don't modify this file. We have a automation pipeline to update this based PR title and PR body content while realeasing.

File: package-lock.json (hunk 1 above)
Comment (audunn) [reply package-lock.json]: Acutely not sure must have been added by a tool like a package manager, perhaps added when I updated azdev. I removed id.

File: src/azure-cli/HISTORY.rst (hunk 2 above)
Comment (audunn) [reply src/azure-cli/HISTORY.rst]: Thanks you for the clarification

File: src/azure-cli/HISTORY.rst [hunk 3]
```diff
@@ -639,7 +639,7 @@ Release History
 * Update error details, update default application template and prompt message.
 * Support app creation/update with the new sku name ST0, ST1, ST2.
 
-**Key Vault**
+**Key Vault**
```
Comment (haroldrandom) [inline src/azure-cli/HISTORY.rst:642]:
```suggestion
**Key Vault**
```

**Comment by audunn** [comment]
please review note this is urgently needed by MSFT for high-profile engagement

**Pull request merged**